products/developertoolset/_common/product.yml

```diff
@@ -1,7 +1,7 @@
 name: Red Hat Developer Toolset
 abbreviated_name: Developer Toolset
 product_logo: portal.png
-current_version: 3.1.0.GA
+current_version: 4.1
```

Comment:
My guess is it's failing because of this line. We're expecting a string and this is a float. You could certainly surround it by quotes, but shouldn't it include a .GA or something like that?